apps/chatbot/src/modules/models.py

```diff
@@ -0,0 +1,79 @@
+import os
+import boto3
+import logging
+from typing import Any, Mapping
+from langchain_aws import BedrockLLM
+from langchain_core.callbacks import AsyncCallbackHandler
+from llama_index.llms.langchain import LangChainLLM
+from llama_index.embeddings.langchain import LangchainEmbedding
+from langchain_aws.embeddings import BedrockEmbeddings
+from dotenv import load_dotenv
+
+load_dotenv()
+
+AWS_ACCESS_KEY_ID = os.getenv('CHB_AWS_ACCESS_KEY_ID', os.getenv('AWS_ACCESS_KEY_ID'))
+AWS_SECRET_ACCESS_KEY = os.getenv('CHB_AWS_SECRET_ACCESS_KEY', os.getenv('AWS_SECRET_ACCESS_KEY'))
```

Comment:
It is preferable to remove the default because it would not work on lambda as tested before
```suggestion
AWS_ACCESS_KEY_ID = os.getenv('CHB_AWS_ACCESS_KEY_ID')
AWS_SECRET_ACCESS_KEY = os.getenv('CHB_AWS_SECRET_ACCESS_KEY')
```